examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+"""
+======================================
+Demonstrate morphing VolSourceEstimate
+======================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose `dipy
+<http://nipy.org/dipy>`_ will be
+used in order to perform the necessary transforms to a fsaverage serving as
+reference space. The example uses parts of the MNE example
+:ref:`LCMV beamformer pipeline
```

Comment:
I wouldn't give it an alternate title, it probably looks fine as:
```
:ref:`sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py`
```

Reply:
check!